src/Symfony/Bridge/Twig/Extension/SecurityExtension.php

```diff
@@ -84,6 +87,19 @@ public function getImpersonatePath(string $identifier): string
         return $this->impersonateUrlGenerator->generateImpersonationPath($identifier);
     }
 
+    public function userIsGranted(UserInterface $user, mixed $role, mixed $object = null, ?string $field = null): bool
```

Comment:
I suggest using `$subject` instead of `$object` to match the naming in the component (nothing forces subjects to be objects)